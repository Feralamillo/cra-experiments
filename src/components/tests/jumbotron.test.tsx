import React from 'react';
import { shallow } from 'enzyme';

import { Jumbotron } from '../jumbotron';

test('renders the component', () => {
  const wrapper = shallow(<Jumbotron />);

  expect(wrapper).toMatchSnapshot();
});
