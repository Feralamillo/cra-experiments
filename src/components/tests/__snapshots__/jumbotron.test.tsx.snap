// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<section
  className="jumbotron text-center mb-0 bg-white"
>
  <div
    className="container"
  >
    <h1
      className="jumbotron-heading"
    >
      Album example
    </h1>
    <p
      className="lead text-muted"
    >
      Something short and leading about the collection belowâ€”its contents, the creator, etc. Make it short and sweet, but not too short so folks don't simply skip over it entirely.
    </p>
    <p>
      <a
        className="btn btn-primary m-2"
        href="!#"
      >
        Main call to action
      </a>
      <a
        className="btn btn-secondary m-2"
        href="!#"
      >
        Secondary action
      </a>
    </p>
  </div>
</section>
`;
